pyunlocbox/functions.py

```diff
@@ -993,6 +1044,74 @@ def _prox(self, x, T):
         return sol
 
 
+class proj_lineq(proj):
+    r"""
+    Projection on the plane argmin_x || x - z||_2 s.t. Ax = b
+
+    This function is the indicator function :math:`i_S(z)` of the set S which
+    is zero if `z` is in the set and infinite otherwise. The set S is defined
+    by :math:`\left\{z \in \mathbb{R}^N \mid z \leq 0 \right\}`.
```

Comment:
Out-of-date too?

Reply:
Fixed

Reply:
Does `M \leq 0` mean symmetric positive semi-definite?

Reply:
positive semi definite. maybe we should use $\succeq$

Reply:
Yeah `\succeq` is better (`\leq` could be negative semi-definite). What about the symmetric part?

Reply:
What about adding `,M=M^T`

Reply:
👍